c_emulator/riscv_sim.c

```diff
@@ -120,6 +120,7 @@ static struct option options[] = {
     {"disable-writable-misa",       no_argument,       0, 'I'},
     {"disable-fdext",               no_argument,       0, 'F'},
     {"mtval-has-illegal-inst-bits", no_argument,       0, 'i'},
+    {"c-hints-expand",              no_argument,       0, 'H'},
```

Comment:
Would rather not have 'H' used as the single character switch,  as it probably should be reserved for hypervisor. I would suggest to only have a long option.

Reply:
Happy to, although if I understand correctly, it will mean some refactoring of this file, as currently all the options have short forms and that's what is selected for in the big switch statement.

Reply:
I think you can use a any number beyond the ascii-table.  Just remember to remove the short form of this option from the `print_usage()` function.

For example,
```c
#DEFINE C_HINTS_EXPAND_OPT 1000

{"c-hints-expand",              no_argument,       0, C_HINTS_EXPAND_OPT},

case C_HINTS_EXPAND_OPT:
     ...
```